Gems/ROS2/Code/Source/Camera/CameraSensor.cpp

```diff
@@ -240,4 +253,51 @@ namespace ROS2
         return "Color";
     };
 
+    CameraRGBDSensor::CameraRGBDSensor(const CameraSensorDescription& cameraSensorDescription)
```

Comment:
Do you need to implement the GetPipelineTypeName() function to return a name different than "Color"?

Reply:
Thanks, It seems to be unnecessary, GetPipelineTypeName is used to create only pipeline instance names. I would not change it in this PR, since it is currently in codebase.